pkg/test/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/anz-bank/sysl-catalog/pkg/test
+
+go 1.14
+
+require (
+	github.com/anz-bank/protoc-gen-sysl v0.0.8 // indirect
+	github.com/anz-bank/sysl v0.97.0
+	github.com/anz-bank/sysl-catalog v1.4.29
+	github.com/sirupsen/logrus v1.6.0
+	github.com/spf13/afero v1.2.2
+	github.com/stretchr/testify v1.5.1
+)
```

Comment:
get rid of this and the test file, 
Or test every dependency we've got, 
It doesn't make sense to me to have another test file within here, especially considering that we're only testing to see wether a local file can be loaded with sysl modules

Reply:
Yup will delete!